pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -121,11 +121,11 @@ func (t *ofTable) ResetStatus() {
 }
 
 // BuildFlow returns FlowBuilder object to help construct Openflow entry.
-func (t *ofTable) BuildFlow(priority uint16) FlowBuilder {
```

Comment:
I don't think this change is necessary, we can pass the priority value (uint16) in caller, right?

Reply:
If not, we need to change its caller like `return PipelineIPClassifierTable.ofTable.BuildFlow(priorityHigh).`. That's a change big.